packages/manager/src/features/CloudPulse/Alerts/ContextualView/AlertInformationActionRow.tsx

```diff
@@ -35,13 +36,30 @@ export const AlertInformationActionRow = (
   const { alert, handleToggle, status = false } = props;
   const { id, label, rule_criteria, service_type, type } = alert;
   const metricThreshold = processMetricCriteria(rule_criteria.rules);
+  const isAccountOrRegionLevelAlert = (alert: Alert) =>
+    alert.scope === 'region' || alert.scope === 'account';
```

Comment:
```suggestion
  const isAccountOrRegionLevelAlert = alert.scope === 'region' || alert.scope === 'account';
```

no need to make it a function as row already in the context of single alert. Just compute it directly